app/Items/page.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import styles from '@/app/ui/style.css';
+
+export default function Page() {
+  return (
+    <div>
+        <div className={styles.shape} />
+        <meta charSet="utf-8" />
+        <link rel="icon" href="/customers/amy-burns.png" />
+        <meta name="viewport" content="width=device-width" />
+        <title>Grocery Store</title>
+        <link href="style.css" rel="stylesheet" type="text/css" />
```

Comment:
Why are you linking the style sheets this way?

Just do:
import '@/app/ui/style.css';

And give the .css file a unique name

Reply:
Feel free to use my pages as examples for the global.css overrides